### ISSUE TITLE
Asking for datasets for reproduction

### ISSUE THREAD

**Issue by jej127** [allenai/clarifydelphi#2]
Hello, I have a question about reproduction of ClarifyDelphi.
According to the source code, two datasets 'data/train_social_chem_with_prefix_t5.jsonl' and 'data/dev_social_chem_with_prefix_t5.jsonl' seem needed.
They are relevant to "args.dataset_train" and "args.dataset_val" in clarifydelphi/Code/arguments.py, respectively.
However, I can find them nowhere. Could you please explain how I can get these datasets?

**Comment by ValentinaPy** [comment]
You can get these train and dev files from the SocialChemistry dataset: https://maxwellforbes.com/social-chemistry/ .
Because t5 uses prefixes, we added the prefix: "question: " to each of the situations. I hope this helps, let me know if you have more questions!

**Comment by jej127** [comment]
Thanks for answer.
For another question, in the line 128 of the code "clarifydelphi/Code/reward.py", a sentence fusion model seems to be loaded with the line "model_dict_fusion = load_model_t5('checkpoint-11000', cuda_devices=[2])".
How can I obtain this model? Could I access the source code to train this model?

It would be even better if I could access the source code to train a question generation model (line 21-23 of the code "clarifydelphi/Code/arguments.py") and an answer generation model (line 127 of the code "clarifydelphi/Code/reward.py") as well.

Thank you.

**Comment by ValentinaPy** [comment]
Both the question generation model and the answer generation model are based on T5, specifically T5-large. To train the models we used the following huggingface script: https://github.com/huggingface/transformers/tree/main/examples/pytorch/summarization
You simply need to specify the train and dev files.
I hope this helped!

**Comment by jej127** [comment]
Thank you for your answer. It is really helpful.
For the last question, how do I train a sentence fusion model in the line 128 of the code "clarifydelphi/Code/reward.py"?
Training this model seems to require texts called Fusion (e.g., "refraining from doing something bad when you’re angry" in the paper), which I think not contained in the SocialChemistry dataset.
The paper states that "We train a model to distill fusion in-context examples obtained from GPT-3 (text-curie-001)", but I will be grateful If I could have more explanation.

Thank you.

**Comment by ValentinaPy** [comment]
Thank you, I uploaded the distilled fusion data to the github so you can use it to train your fusion model: https://github.com/allenai/clarifydelphi/tree/main/Data/fusion

**Comment by jej127** [comment]
Thank you. This will be helpful a lot.
